.github/workflows/production.yml

```diff
@@ -0,0 +1,60 @@
+name: production-deployment
+on:
+  push:
+    branches: [ dev ]
```

Comment:
looks good for now, eventually - we can switch this trigger to [ main ] or add a dev config.

Reply:
We're only working off of dev right now so I figured this is all we would need